src/components/InputFields.resi

```diff
@@ -43,16 +43,16 @@ let selectInput: (
   ~fixedDropDownDirection: SelectBox.direction=?,
   ~customButton: React.element=?,
   ~buttonType: Button.buttonType=?,
-  ~dropdownCustomWidth: string=?,
+  ~dropdownCustomWidth: Js.String2.t=?,
   ~allowButtonTextMinWidth: bool=?,
-  ~setExtSearchString: (string => string) => unit=?,
-  ~textStyleClass: string=?,
+  ~setExtSearchString: ('b => string) => unit=?,
```

Comment:
can we avoid using generic type

Reply:
These files are generated by rescript extension.